.travis.yml

```diff
@@ -1,21 +1,47 @@
 language: php
+
+services:
+  - elasticsearch
+
 php:
-  - "5.5"
-  - "5.4"
-  - "5.3"
+  - 7.0
+  - 5.6
+  - 5.5
+  - 5.4
+  - 5.3
+  - hhvm
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
+  include:
+    - php: 5.6
+      env: ES_VERSION="1.4.4"
+    - php: 5.6
+      env: ES_VERSION="1.3.9"
+    - php: 5.6
+      env: ES_VERSION="1.2.4"
+    - php: 5.6
+      env: ES_VERSION="1.1.2"
+    - php: 5.6
+      env: ES_VERSION="1.0.3"
+
 env:
-  - ES_VERSION=0.90.0 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.1 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.2 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.3 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.4 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.5 ES_TEST_HOST=http://localhost:9200
+  - ES_VERSION="1.5.1"
+  - ES_TEST_HOST=http://localhost:9200
+
+before_install:
+  - curl -O https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-$ES_VERSION.deb && sudo dpkg -i --force-confnew elasticsearch-$ES_VERSION.deb
```

Comment:
They currently use the official Debian package for 1.4.0 (the version being used can be found in https://github.com/travis-ci/travis-cookbooks/blob/master/ci_environment/elasticsearch/attributes/default.rb)

Any update to newer versions would then be announced on http://docs.travis-ci.com/user/build-environment-updates/
The latest ES update was 5 months ago: http://docs.travis-ci.com/user/build-environment-updates/2014-12-09/ They haven't updated it to newer versions since then in following updates (it could be requested though). Env updates will only happen every 2 months anyway

Regarding getting the version, isn't there a way to get the version of ES from the installed ES itself (something like ``elasticsearch --version`` ?) If no, you would have to update the version manually after Travis updates impacting elasticsearch